types/index.d.ts

```diff
@@ -1401,7 +1401,7 @@ declare namespace Knex {
   // Chainable interface
   //
 
-  interface ChainableInterface<T = any> extends Promise<T> {
+  interface ChainableInterface<T = any> extends Pick<Promise<T>, "then" | "catch" | "finally"> {
```

Comment:
Does promise have "finally" method? I thought it was just a bluebird thing.

Reply:
The Typescript`lib.es2018.promise.d.ts` definition added `finally`

https://github.com/microsoft/TypeScript/blob/234680851bbbe83dd6bcd8aac165a57adbf586e2/src/lib/es2018.promise.d.ts#L11

I don't follow the whole feature ratification process.  I just know `finally` works in Node 12.